### DISCUSSION TITLE
lesson 5: UnicodeDecodeError: 'utf-8' codec can't decode byte 0xb6 in position 260: invalid start byte

### DISCUSSION THREAD

**Discussion by ace-brown** [smartcontractkit/full-blockchain-solidity-course-py#203] [Q&A] [answered]
hi guys. I was trying to deploy the print("hello world!") in the min 04:34:00 of the tutorial. But i got the following error. Can anybody help me with this?
![Capture](https://user-images.githubusercontent.com/79329055/137517150-6b6abd17-86c6-4ad5-89c4-31e5ae39ee51.JPG)

**Comment by PatrickAlphaC** [comment, marked as answer]
Could you please copy-paste the text of the error as opposed to the screenshot?

**Reply by ace-brown** [reply]
Thank a lot Patrick. I managed to solve this error. It was a typo by the way. it turns out that in the .env file "3" from web3 has been missing.
export WEB3_INFURA_PROJECT_ID=7c649f11ba9543b7813eb3ebdbc75051 

BTW, would you please take a look at this error for me: #213 
I literally got stuck here and can't proceed further into the course.
